src/components/WebexAvatar/README.md

```diff
@@ -1,9 +1,22 @@
 # Webex Avatar Component
+Webex avatar component displays the avatar and status of a _Webex Teams_ user.
 
-Webex avatar component displays the avatar of a _Webex Teams_ user with a given `personID` and `adapter` object.
+![Webex Avatar](./WebexAvatar.png)
```

Comment:
if we had more than one `png`s, I prefer to have a `imgs/files` directory for each component.